controllers/front/notify.php

```diff
@@ -318,9 +324,9 @@ public function getMerchantOrder()
     private function getPagantisOrderId()
     {
         try {
-            $this->pagantisOrderId= Db::getInstance()->getValue(
-                'select order_id from '._DB_PREFIX_.self::ORDERS_TABLE.' where id = '.(int)$this->merchantCartId
-            );
+            $sql = 'select order_id from ' . _DB_PREFIX_.self::ORDERS_TABLE .' where id = '
+                .(int)$this->merchantCartId . ' and token = \'' . $this->token . '\'';
```

Comment:
scaping token parameter? why not id? Remove both?

Reply:
I'm preserving simple quote for string sql parameter not scaping